docs/changehistory/NextVersion.md

```diff
@@ -5,7 +5,54 @@ publish: false
 
 ## New Settings UI Features
 
-The @bentley/ui-core package has added the [SettingsManager]($ui-core) class that allows any number of [SettingsProvider]($ui-core) classes to be registered. These providers provide [SettingsTabEntry]($ui-core) definitions used to populate the [SettingsContainer]($ui-core) UI component with setting pages used to manage application settings. These new classes are marked as beta in this release and are subject to minor modifications in future releases.
+The @bentley/ui-core package has added the [SettingsManager]($ui-core) class that allows any number of [SettingsProvider]($ui-core) classes to be registered. These providers provide [SettingsTabEntry]($ui-core) definitions used to populate the [SettingsContainer]($ui-core) UI component with setting pages used to manage application settings. The [SettingsModalFrontstage]($ui-framework) can be provided to the [BackstageComposer]($ui-framework) to automatically display a `Settings` option in the backstage menu if `SettingsProviders` are registered with the `SettingsManager`. Below is a simple example of providing a `Settings` stage.
+
+```tsx
+export function AppBackstageComposerComponent({ userInfo }: AppBackstageComposerProps) {
+  const [backstageItems] = React.useState(() => {
+    return [
+      BackstageItemUtilities.createStageLauncher(ViewsFrontstage.stageId, 100, 10, IModelApp.i18n.translate("SampleApp:backstage.viewIModel"), IModelApp.i18n.translate("SampleApp:backstage.iModelStage"), `svg:${stageIconSvg}`),
+      SettingsModalFrontstage.getBackstageActionItem (400, 10),
+    ];
+  });
+
+  return (
+    <BackstageComposer items={backstageItems}
+      header={userInfo && <UserProfileBackstageItem userInfo={userInfo} />}
+    />
+  );
+}
+```
+
+These new classes are marked as beta in this release and are subject to minor modifications in future releases.
+
+### Add Settings Page to set Quantity Formatting Overrides
+
+The [QuantityFormatSettingsPanel]($ui-framework) component has been added to the @bentley/ui-framework package to provide the UI to set both the [PresentationUnitSystem]($presentation-common) and formatting overrides in the [QuantityFormatter](imodeljs-frontend). This panel can be used in the new [SettingsContainer]($ui-core) UI component. The function `getQuantityFormatsSettingsManagerEntry` will return a [SettingsTabEntry]($ui-core) for use by the [SettingsManager]($ui-core). Below is an example of registering the `QuantityFormatSettingsPanel` with the `SettingsManager`.
```

Comment:
`[QuantityFormatter](imodeljs-frontend)` is missing the `$`

Reply:
fixed